doc/man/scons.xml

```diff
@@ -2612,9 +2612,20 @@ env.SomeTool(targets, sources)
 <title>Builder Methods</title>
 
 <para>You tell &scons; what to build
-by calling <firstterm>Builders</firstterm>, functions which know to take a
-particular action to produce a particular result type
-when given source files of a particular type. &scons;
+by calling <firstterm>Builders</firstterm>,
+functions which take
+particular actions to produce a particular result type
```

Comment:
action(s) ?

Reply:
made this change (and, naturally, couldn't resist a few others)